tools/metamorpheus/macros.xml

```diff
@@ -0,0 +1,712 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.0.316</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">metamorpheus</requirement>
+            <yield/>
+        </requirements>
+    </xml>
+
+    <token name="@DEFAULT_COMMON_TOML@">
```

Comment:
What is the idea of this toml file? There seems to be some intersections with the others? Should the parameters be configurable?